client/js/gdpr/chart.js

```diff
@@ -123,6 +127,7 @@ export class GdprChart extends HTMLElement {
             key,
           },
           bubbles: true,
+          composed: true,
```

Comment:
is it possible to add a comment somewhere about why adding `composed` is important?

Reply:
I could do that, but I think that since `composed` essentially only has one function (to allow events to cross the shadow root boundary) it should be up to the developer to understand how custom events work.